Sources/AsyncCollections/CompactMap.swift

```diff
@@ -0,0 +1,123 @@
+
+extension Sequence where Element: Sendable {
+    /// Returns an array containing the non-`nil` results of calling the given
+    /// transformation with each element of this sequence.
+    ///
+    /// Use this method to receive an array of non-optional values when your
+    /// transformation produces an optional value.
+    ///
+    /// The closure calls are made serially. The next call is only made once the previous call
+    /// has finished. Returns once the closure has run on all the elements of the Sequence
+    /// or when the closure throws an error.
+    /// - Parameter transform: An async  mapping closure. transform accepts an
+    ///     element of this sequence as its parameter and returns a transformed value of
+    ///     the same or of a different type.
+    /// - Returns: An array containing the transformed elements of this sequence.
+    public func asyncCompactMap<T>(_ transform: @Sendable (Element) async throws -> T?) async rethrows -> [T] {
+        var result = ContiguousArray<T>()
+
+        var iterator = self.makeIterator()
+
+        // Add remaining elements, if any.
+        while let next = iterator.next() {
+            if let element = try await transform(next) {
+                result.append(element)
+            }
+        }
+        return Array(result)
+    }
+
+    /// Returns an array containing, in order, the non-`nil` results of calling the given
+    /// transformation with each element of this sequence.
+    ///
+    /// Use this method to receive an array of non-optional values when your
+    /// transformation produces an optional value.
+    ///
+    /// This differs from `asyncCompactMap` in that it uses a `TaskGroup` to run the transform
+    /// closure for all the elements of the Sequence. This allows all the transform closures
+    /// to run concurrently instead of serially. Returns only when the closure has been run
+    /// on all the elements of the Sequence.
+    /// - Parameters:
+    ///   - priority: Task priority for tasks in TaskGroup
+    ///   - isIncluded: An async closure that takes an element of the
+    ///   sequence as its argument and returns a Boolean value indicating
+    ///   whether the element should be included in the returned array.
+    /// - Returns: An array of the elements that `isIncluded` allowed.
+    public func concurrentCompactMap<T: Sendable>(priority: TaskPriority? = nil, _ isIncluded: @Sendable @escaping (Element) async throws -> T?) async rethrows -> [T] {
```

Comment:
Any reason the closure is called `isIncluded` here but `transform` above. The stdlib uses `transform`.

Reply:
I know its a minor thing, but why did you resolve this without comment and similarly below

Reply:
ah, didn't think you'd expect a comment since i applied the requested changes 😅

Reply:
I don't see the changes. I'm looking on my phone so maybe I've missed them.

Reply:
Opps sorry, apparently forgot to actually push the changes. Will do in a few mins.